src/titiler/mosaic/setup.py

```diff
@@ -9,8 +9,8 @@
     "titiler.core",
     "cogeo-mosaic>=3.0,<3.1",
     "mercantile",
-    "pystac-client~=0.1.1",
-    "stac_pydantic",  # inherit version from pystac-client
+    "pystac-client~=0.6.0",
```

Comment:
pystac-client 0.1.1 only supported the stac api beta.1 conformance classes, and would fail if anything newer is present